sdk/iot/tests.yml

```diff
@@ -0,0 +1,10 @@
+trigger: none
+
+stages:
+  - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      AllocateResourceGroup: 'false'
+      BuildTargetingString: $(BuildTargetingString)
+      ServiceDirectory: iot
+      EnvVars:
+        TEST_MODE: 'RunLiveNoRecord'
```

Comment:
Normally we'd want to run recordings in the pipeline. The live tests happen locally and test recordings would be checked-in.

Reply:
Recordings have not yet been generated due to a what appears to be a bug in the testing infrastructure. This will be changed once recordings can be generated.